backend/climateconnect_api/utility/email_setup.py

```diff
@@ -272,3 +278,117 @@ def remove_contact_from_list(contact_id, list_id):
         ]
     }
     mailjet_api.contact_managecontactslists.create(id=contact_id, data=data)
+
+
+def create_variables_for_weekly_recommendations(language_code: str, project_ids: List = [], organization_ids: List = [], idea_ids: List = []):
+    # edit the image and url path for production
+    # todo idea has no url 
+   
+    main_page = {
+        "en": "https://climateconnect.earth/en/browse",
+        "de": "https://climateconnect.earth/de/browse",
+    }
+    project_translation = {
+        "en": "Project",
+        "de": "Projekt",
+    }
+    organization_translation = {
+        "en": "Organization",
+        "de": "Organisation",
+    }
+    idea_translation = {
+        "en": "Idea",
+        "de": "Idee",
+    }
+
+    entities = []
+    for project_id in project_ids:
+        project = Project.objects.select_related('loc').get(id=project_id)
+        project_template = {
+            "type": project_translation.get(language_code, "en"),
+            "name": project.name,
+            "imageUrl": (settings.ALLOWED_HOSTS[6] + project.thumbnail_image.url) if project.thumbnail_image else '',
```

Comment:
settings.ALLOWED_HOSTS[6] looks very dangerous incase we ever change that array. Can't we use `settings.FRONTEND_URL` instead?